tool/README.md

```diff
@@ -0,0 +1,25 @@
+# SECBENCH scripts
+
+These are the scripts that were utilized to conduct the research.
+To make use of these scripts with minimal modifications, you would need to have:
+- github API key
+- google cloud account
+- redis cache
```

Comment:
I think I'll have to change the master and remove the google cloud and redis dependencies. They do not make sense at all. I just used them as a requirement for my master thesis.

Reply:
#4